automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -272,7 +262,7 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
         // http://httr.r-lib.org//index.html
 
         // it may take a little while to install
-        val installTimeout = 2 minutes
+        val installTimeout = 2.minutes
```

Comment:
To get rid of the compiler warning...it didn't feel worth it to add a `postfixOps` import.